rbd.c

```diff
@@ -1026,8 +1026,9 @@ static int tcmu_rbd_open(struct tcmu_device *dev, bool reopen)
 	char *config, *dev_cfg_dup;
 	struct tcmu_rbd_state *state;
 	uint32_t max_blocks, unmap_gran;
-	int ret;
 	char buf[128];
+	char *savedptr;
```

Comment:
I know it doesn't matter for `strtok_r` but it's a good idea to initialize anyway as some compilers and static checkers would complain.
```suggestion
	char *saveptr = NULL;
```

Reply:
Done.